app/views/candidates/registrations/personal_informations/_form.html.erb

```diff
@@ -37,6 +37,14 @@
                 readonly: personal_information.read_only,
                 disabled: personal_information.read_only %>
 
+          <p>
+            Your details are protected under the terms of our <%= link_to 'privacy notice', 'https://schoolexperience.education.gov.uk/privacy_policy' %>.
```

Comment:
We normally open the privacy policy in a new tab so that people can read it alongside the form?